lib/event-handlers.js

```diff
@@ -1,20 +1,21 @@
-var props = require('../resources/properties');
+'use strict';
+const props = require('../resources/properties');
 
-var registerEventHandlers = function (eventHandlers) {
-  eventHandlers.onSessionStarted = function (sessionStartedRequest, session) {
-    console.log('MetroTransit onSessionStarted requestId: ' + sessionStartedRequest.requestId +
-        ', sessionId: ' + session.sessionId);
+const registerEventHandlers = (eventHandlers) => {
```

Comment:
Side note, we aren't consistent with how we export the functions. See api-wmata compared to this one. in api-wmata, we export it inline, but here we define and export at the end.